elevenclock/lang/lang_cs.py

```diff
@@ -11,21 +11,21 @@
 # Meke special attention to elements like ":", etc.
 
 lang_3_3 = {
-    "Custom format rules:": "",
+    "Custom format rules:": "Pravidla vlastního formátu:",
     "Any text can be placed here. To place items such as date and time, please use the 1989 C standard. More info on the following link": "",
     "Python Date and time values": "",
     "To disable the zero-padding effect, add a # in bethwwn the % and the code: non-zero-padded hours would be %#H, and zero-padded hours would be %H": "", # Here please don't modify the %H and %#H values
```

Comment:
"bethwwn"? :)

@martinet101

Reply:
Yes, it was between, it has been fixed in the English translation, but I have to replace this on the entire code.